packages/terra-application-navigation/README.md

```diff
@@ -0,0 +1,24 @@
+# Terra Application Navigation
+
+The Terra Application Navigation is a responsive, themeable layout for building applications. ApplicationNavigation is comprised of selectable navigation and utility elements with selection callbacks allowing for versatile implementations of state and routing.
+
+[![NPM version](https://badgen.net/npm/v/terra-application-navigation)](https://www.npmjs.org/package/terra-application-navigation)
+[![Build Status](https://badgen.net/travis/cerner/terra-framework)](https://travis-ci.org/cerner/terra-framework)
+
+- [Getting Started](#getting-started)
+- [Documentation](https://github.com/cerner/terra-framework/tree/master/packages/terra-application-navigation/docs)
+- [LICENSE](#license)
+
+## Getting Started
+
+- Install from [npmjs](https://www.npmjs.com): `npm install terra-application-navigation`
```

Comment:
Add this comment after this line:

```
If you rename this file to README.md and add the following comment, the peer dependency info for this package will be auto-generated when the docs:peer-dependencies script is run:

```
<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
<!-- AUTO-GENERATED-CONTENT:END -->
```

Reply:
Updated.